### PR TITLE
Clear tscache on postinstall

### PR DESCRIPTION
**Pull request by buoyad**
also remove pegjs

### CODE REVIEW

File: shared/package.json [hunk 1]
```diff
@@ -272,12 +272,12 @@
     "msgpack": "git://github.com/keybase/nullModule",
     "net": "git://github.com/keybase/nullModule",
     "null-loader": "0.1.1",
-    "pegjs": "0.10.0",
     "prettier": "1.17.1",
     "randexp": "0.5.3",
     "react-hot-loader": "4.11.0",
     "react-test-renderer": "16.8.4",
     "redux-logger": "3.0.6",
+    "rimraf": "3.0.0",
```
Comment (chrisnojima) [inline shared/package.json]: just use 2.2.8 so we don't have any new code

Comment (buoyad) [reply shared/package.json]: Making it 2.6.3 which we also already have makes fewer lockfile changes, i'll use that one

File: shared/desktop/yarn-helper/index.tsx [hunk 2]
```diff
@@ -109,6 +111,23 @@ const decorateInfo = info => {
   return temp
 }
 
+const warnFail = path => () => console.warn(`Error cleaning tscache ${path}, tsc may be inaccurate.`)
+const clearTSCache = () => {
+  const tsOutDir = path.resolve(__dirname, '..', '..', '.tsOuts')
```
Comment (chrisnojima) [inline shared/desktop/yarn-helper/index.tsx]: you can readDirSync and just blow away everything other than README.md

Comment (buoyad) [reply shared/desktop/yarn-helper/index.tsx]: `rimraf` supports globs, so we could have it do `.tsOuts/.tsOut*`, wdyt?

Comment (chrisnojima) [reply shared/desktop/yarn-helper/index.tsx]: even better

Comment (buoyad) [reply shared/desktop/yarn-helper/index.tsx]: Done, thanks

**Pull request merged**